src/euler3d_mappedGrid.f90

```diff
@@ -0,0 +1,679 @@
+MODULE mg
+
+  REAL(kind=8), PARAMETER :: pi=3.1415926535897932385,twopi=2*pi
+  REAL(kind=8), PARAMETER :: rEarth=637120000.0
```

Comment:
I am somewhat curious about your constant here, we use a value of `6367.5e3` in meters for the radius of the earth.  Also note that both of these parameters are not being declared as double precision numbers.